core/kazoo_amqp/Makefile

```diff
@@ -1,6 +1,7 @@
 ROOT = ../..
 PROJECT = kazoo_amqp
 
+SOURCES = src/gen_listener.erl src/*.erl $(if $(wildcard src/*/*.erl), src/*/*.erl)
```

Comment:
Good point; think I just copied it over from kz.mk because without it the api modules weren't getting compiled.